validator/src/main.rs

```diff
@@ -615,12 +615,12 @@ pub fn main() {
         .arg(
             clap::Arg::with_name("limit_ledger_size")
                 .long("limit-ledger-size")
-                .value_name("SLOT_COUNT")
```

Comment:
@mvines are we ok changing the meaning of this to be based on shred count for 1.0?

Reply:
I don't see why not.  I'll need to adjust this setting on our nodes since we don't use the default here, but we don't advise anything but the default for all the validators so this change ought to be mostly unnoticed